Contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to DeliverEase
```

Comment:
Maybe you can bold or italicize important words or phrases. That would increase the readability tremendously